src/index.ts

```diff
@@ -2,3 +2,52 @@
 // Node module: loopback-next-extension-starter
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
+
+import { Application, get } from '@loopback/core';
+import { transactionIdHeader } from './decorators/transaction-id';
+import { inject } from '@loopback/context';
+
+
+class MyController {
+	@get('/')
+	@transactionIdHeader()
```

Comment:
I also tried injecting `http.request` into transactionIdHeader() ... but that didn't work for obvious reasons.